actionners/aws/lambda/lambda.go

```diff
@@ -0,0 +1,131 @@
+package lambda
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/Falco-Talon/falco-talon/internal/aws/client"
+	"github.com/Falco-Talon/falco-talon/internal/events"
+	"github.com/Falco-Talon/falco-talon/internal/rules"
+	"github.com/Falco-Talon/falco-talon/utils"
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+	"github.com/aws/aws-sdk-go-v2/service/lambda/types"
+	"github.com/mitchellh/mapstructure"
+	"strconv"
+)
+
+func Action(action *rules.Action, event *events.Event) (utils.LogLine, error) {
+
+	lambdaClient := client.GetAWSClient().GetLambdaClient()
+	parameters := action.GetParameters()
+
+	lambdaConfig, err := NewLambdaConfig(parameters)
+	if err != nil {
+		return utils.LogLine{
+				Objects: nil,
+				Error:   err.Error(),
+				Status:  "failure",
+			},
+			err
+	}
+
+	objects := map[string]string{
+		"name":    lambdaConfig.AWSLambdaName,
+		"version": lambdaConfig.AWSLambdaAliasOrVersion,
+	}
+
+	payload, err := json.Marshal(event.OutputFields)
```

Comment:
Why do you send only the output fields? I think it could be better to send the whole payload, to allow the lambda to get access to the falco rule name, the priority, the source, the tags, ...

Reply:
I thought that if we can manage the output fields from Talon users would be able to call existing functions just by changing the way output_fields are sent. 

If we sent the entire payload, existing functions would need to be changed to inspect **event.output_fields**, since the event itself will be the payload.

For instance, if I trigger the aws:lambda action with this event:
```
curl --location 'http://localhost:2803/' \
--header 'Content-Type: application/json' \
--data '{
    "output": "Test invoke lambda",
    "priority": "Warning",
    "Rule": "Test invoke lambda",
    "time": "2019-05-17T15:31:56.746609046Z",
    "output_fields": {
        "test": "true",
        "shoudFail": "true"
    }
}'
```

I'm essentially passing this event to my lambda function:
```
{ shoudFail: 'true', test: 'true' }
```

We can easily forward the entire event though, WDYT?

Reply:
I think we should forward the full falco event, to avoid the lambda to have any restriction. My idea is to append some fields to the `output_fields`:

```json
{
    "uuid": "12439410-df5a-4e45-9cbf-6efa577f930d",
    "output": "12:52:28.563070379: Notice A shell was spawned in a container with an attached terminal (evt_type=execve user=root user_uid=0 user_loginuid=-1 process=bash proc_exepath=/usr/bin/bash parent=containerd-shim command=bash terminal=34817 exe_flags=EXE_WRITABLE container_id=572c5475cbe5 container_image=docker.io/library/debian container_image_tag=latest container_name=cncf k8s_ns=default k8s_pod_name=cncf-55696bc998-ms625)",
    "priority": "Notice",
    "rule": "Terminal shell in container",
    "time": "2024-03-22T12:52:28.563070379Z",
    "source": "syscall",
    "output_fields": {
	...
	"falco-talon.rule": "xxxxx",
	"falco-talon.action": "xxxx",
	"falco-talon.actionner": "xxxx",
	"falco-talon.continue: "xxxx",
	"falco-talon.dry-run": "xxxx",
	"falco-talon.action.ignore_errors": "xxxx",
	"falco-talon.action.parameters": {
		"xxxx": "xxxx",
		"xxxx": "xxxx"
	} 
    },
    "hostname": "falco-mkj9v",
    "tags": [
        "T1059",
        "container",
        "maturity_stable",
        "mitre_execution",
        "shell"
    ]
}
```

wdyt?

Reply:
I think this is good, more context to lambda is better. I'll change it to forward the entire event.

Reply:
Done